bioimageio/spec/utils/maybe_convert_to_v0_3.py

```diff
@@ -154,11 +153,19 @@ def convert_model_v0_3_1_to_v0_3_2(data: Dict[str, Any]) -> Dict[str, Any]:
 
         weights_entry["authors"] = [{"name": name} for name in weights_entry["authors"]]
         authors_update = future.get("weights", {}).get(weights_format, {}).get("authors")
-        if authors_update is None:
-            authors_update = [{"affiliation": "<unknown>"}] * len(data["authors"])
-
-        for a, u in zip(weights_entry["authors"], authors_update):
-            a.update(u)
+        if authors_update is not None:
+            for a, u in zip(weights_entry["authors"], authors_update):
+                a.update(u)
+
+    # documentation (if not a local relative md file, pretend content is in local 'auto_converted_documentation.md')
+    class DocSchema(Schema):
+        doc = schema.Model.documentation
+
+    if DocSchema().validate({"doc": data["documentation"]}):
+        data["config"] = data.get("config", {})  # make sure config exists
```

Comment:
Why is "config" involved here? I'd assume that we just put the content of the "documentation" field into `AUTO_CONVERTED_DOCUMENTATION_FILE_NAME`?

Reply:
that would not pass the schema, so I thought `config` would be a good place too keep it.

Reply:
Yes, but why are you adding 
```python
data["config"][AUTO_CONVERTED_DOCUMENTATION_FILE_NAME] = data["documentation"]
```
in the first place?
Is it just to keep track of the fact that the documentation has been auto generated?

Reply:
to save the original content of `data["documentation"]` (that would need to be written to the AUTO_CONVERTED_DOCUMENTATION_FILE_NAME file)

Reply:
as the converter only has the yaml content as dict it cannot write the file as it wouldn't know where to.

Reply:
Ok, makes sense. Could you sum this up in a comment though? This is not obvious from the code alone.